package.json

```diff
@@ -48,21 +51,25 @@
     "babel-preset-es2015": "^6.6.0",
     "babel-runtime": "^6.22.0",
     "babelify": "^7.3.0",
+    "canvas": "^1.6.5",
```

Comment:
@wimrijnders What do we need this for? This module [requires Cairo](https://github.com/Automattic/node-canvas#canvas-graphics-api-backed-by-cairo) wich is an unacceptable external dependency. Since this I can not install vis anymore on linux 😭

Reply:
`canvas` is used to run unit tests on `Network`. Why is `cairo` an unacceptable external dependency?

Reply:
cairo can not be installed via npm and therefor adds a OS-specific dependency. How shozuld the user (like me) know that he has to install g++ and libgif-dev? `npm install` now just fails. I personally think this is not acceptable!
We should not have dependencies other that `npm` (and maybe `git`) as far as I'm concerned.
The least thing would be to document this in the README.

Reply:
I've created an issue for that: #3515 Lets discuss there, what the best way is to solve this.